src/app/components/start/start.component.scss

```diff
@@ -1,10 +1,11 @@
 #start-container {
-    position: absolute;
-    top: 0px;
-    width: 100%;
-    height: 100%;
-    display:flex;
-    align-items:center;
-    justify-content:center;
-    flex-direction: column;
-  }
\ No newline at end of file
+  position: absolute;
```

Comment:
@jeroenoomsNL do you use a specific linting tool for css? If we all use the same, it will improve code readability.

Reply:
I don't use linting for CSS, only prettier for formatting